package.json

```diff
@@ -15,47 +26,38 @@
     "url": "https://github.com/{github_username}/{github_slug}/issues"
   },
   "homepage": "https://github.com/{github_username}/{github_slug}#readme",
-  "scripts": {
-    "build": "env OPTIMIZE=true webpack",
-    "clean": "rm -rf public",
-    "copy:manifest": "install -m0644 package.json README.md LICENSE public",
-    "deploy": "git-directory-deploy --username {github_username} --email {github_email} --directory public/ --branch build --repo=https://$GITHUB_TOKEN@github.com/{github_username}/{github_slug}.git",
-    "tx": "tx pull --all || true",
-    "postbuild": "npm run copy:manifest",
-    "prebuild": "npm-run-all clean tx",
-    "prewatch": "npm run clean",
-    "test": "true",
-    "watch": "webpack --display-modules --display-chunks --watch"
-  },
   "devDependencies": {
-    "autoprefixer": "^6.5.3",
-    "babel-core": "^6.18.2",
-    "babel-loader": "^6.2.8",
-    "babel-plugin-transform-runtime": "^6.15.0",
-    "babel-preset-es2015": "^6.18.0",
-    "babel-runtime": "^6.18.0",
+    "autoprefixer": "^6.5.0",
+    "babel-core": "^6.0.0",
+    "babel-loader": "^6.0.0",
+    "babel-plugin-transform-runtime": "^6.0.0",
+    "babel-preset-es2015": "^6.0.0",
```

Comment:
What do you think of using [babel-preset-env](https://github.com/babel/babel-preset-env) instead of babel-preset-es2015?